README.md

```diff
@@ -1,9 +1,53 @@
-# ml4ir: Machine Learning libraries for Information Retrieval
+# ml4ir: Machine Learning for Information Retrieval
+**Quickstart &rarr;** **[ml4ir Read the Docs](https://ml4ir.readthedocs.io/en/latest/)** | **[ml4ir pypi](https://pypi.org/project/ml4ir/)** | **[python ReadMe](python/)**
 
-## Getting Started
 
-[Quickstart](python/README.md)
+ml4ir is an open source library for training and deploying deep learning models for search applications. ml4ir is built on top of **python3** and **tensorflow 2.x** for training and evaluation. It also comes packaged with scala utilties for **JVM inference**.
+
+ml4ir is designed as modular subcomponents which can each be combined and customized to build a variety of search ML models such as:
+* Learning to Rank
+* Query Auto Completion
+* Document Classification
+* Query Classification
+* Named Entity Recognition
+* Top Results
+* Query2SQL
+* *add your application here*
+  
+![ml4ir](python/docs/source/_static/ml4ir.png)
+
+
+## Motivation
+Search is a complex data space with lots of different types of ML tasks working on a combination of structured and unstructured data sources. There is no single library that
```

Comment:
"There existed no..." or "We found no" or "There was not found any" (ie past tense since ml4ir exists now :) )